src/global/_variables.css

```diff
@@ -31,7 +31,6 @@
   --font-color-heading: var(--brand-black);
   --background-body: var(--brand-white);
   --background-content: var(--brand-grey-golden);
-  --background-body-transparent: rgba(249, 248, 246, 0.85);
```

Comment:
suggestion(non-blocking): can we use a global color here? Maybe add this color for global usage. Consistency might be improved.